bind-operator/src/bind.py

```diff
@@ -298,27 +300,37 @@ def _zones_to_files_content(
                 mailbox=config["mailbox"],
             )
 
-            # By default, we hide the active unit.
-            # So only the standbies are used to respond to queries and receive NOTIFY events
-            # If we have no standby unit (a single unit deployment)
-            # then use current unit IP instead
-            ip_list: list[pydantic.IPvAnyAddress] = topology.standby_units_ip or [
-                topology.current_unit_ip
-            ]
+            # If an public ip is configured, we use it for our NS records
```

Comment:
I remember that we discussed about that and for the secondary dns charm, this part is a little bit different:
```
            public_ips = self.topology.dump().public_ips
            if not public_ips:
                logger.debug("Public ips not set, using units ip")
                public_ips = self.topology.dump().units_ip
```

Should we consider again setting the default in the topology if this will be valid for all dns charms?